### PR TITLE
Initial readme

### PR DESCRIPTION
**Pull request by danielhuppmann**
This PR adds a license and an initial readme.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2021, Zebedee Nicholls, Daniel Huppmann
```
Comment (znicholls) [inline LICENSE]:
```suggestion
Copyright (c) 2021, Contributors
```

Could seem more inviting to newcomers?

Comment (znicholls) [reply LICENSE]: Can we add something like 'BSD 3-Clause license' at the top so people know what they're looking at straight away?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright notice,
+      this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright notice,
+      this list of conditions and the following disclaimer in the documentation
+      and/or other materials provided with the distribution.
+    * Neither the name of {{ project }} nor the names of its contributors
```
Comment (znicholls) [inline LICENSE]:
```suggestion
    * Neither the name of AR6 WG1 Data Compilation, OpenSCM nor the names of its contributors
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,37 @@
-# AR6-WG1-Data-Compilation
-Compilation of data from AR6 WG1
+# Data Compilation & Figures for the IPCC AR6 WG1
```
Comment (znicholls) [inline README.md]:
```suggestion
# Data Compilation & Figures from the IPCC AR6 WG1
```

Might be clearer given we didn't make anything for IPCC, we're just using outputs

File: README.md [hunk 4]
```diff
@@ -1,2 +1,37 @@
...
+
+### Data resources
+
+This repository compiles the data into a uniform, csv-based data format
+following the standard established by the Integrated Assessment Modeling Consortium
+([IAMC](https://www.iamconsortium.org)) and used by IPCC WG3.
+
+The format used in this repository is directly compatible with
+the **scmdata** and **pyam** Python packages (see dependencies below),
+but can be easily read with Excel or scripts written on other programming languages.
```
Comment (znicholls) [inline README.md]:
```suggestion
but can be easily read with Excel or scripts written in other programming languages.
```

Comment (znicholls) [reply README.md]: Maybe add something like:

The processed data is stored in `data/processed` (see also `data/README.md`). For complete transparency and reproducibility, all steps to compile the processed data from the raw data are also included in this repository (not true yet, but will be true once we start adding stuff). We will add instructions about how to compile the processed data from the raw data in future pull requests.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,37 @@
-# AR6-WG1-Data-Compilation
-Compilation of data from AR6 WG1
+# Data Compilation & Figures for the IPCC AR6 WG1
+
+© 2021 Zebedee Nicholls & Daniel Huppmann; licensed under the [BSD 3-Clause License](LICENSE).
```
Comment (znicholls) [inline README.md]: As above re copyright to contributors rather than us

**Review by znicholls — APPROVED** [review]
Beauty, few suggestions

File: LICENSE (hunk 1 above)
Comment (danielhuppmann) [reply LICENSE]: This is copied straight from the [GitHub license templates](https://github.com/licenses/license-templates/blob/master/templates/bsd3.txt), and the summary and name will be shown as a summary header, same as on [scmdata](https://github.com/openscm/scmdata/blob/master/LICENSE).

Not sure if the automated recognition of the license would still work if we deviate from the standard license text.

File: README.md (hunk 4 above)
Comment (danielhuppmann) [reply README.md]: I was planning on adding another section on the repo structure and the data folders together with the first PR with actual content.

File: README.md (hunk 5 above)
Comment (danielhuppmann) [reply README.md]: Added an explicit list of contributors

File: LICENSE (hunk 1 above)
Comment (znicholls) [reply LICENSE]: Ok cool let's leave it then

File: README.md (hunk 4 above)
Comment (znicholls) [reply README.md]: Sounds good, some inspo text for you then

File: README.md (hunk 5 above)
Comment (znicholls) [reply README.md]: 👍

**Pull request merged**